File: solution/Sol1.java
package solution;

import java.util.Arrays;

public class Sol1 {
    public static void main(String[] args) {
        Solution solution = new Solution();

        int[] nums = new int[]{2, 7, 11, 13};
        int target = 9;
        System.out.println("Input: nums:" + Arrays.toString(nums));
        System.out.println("Input: target:" + String.valueOf(target));

        int[] answers =  solution.twoSum(nums, target);
        System.out.println("Output: answers:" + Arrays.toString(answers));
    }
}

class Solution {
    public int[] twoSum(int[] nums, int target) {
        int answer_list[] = new int[2];
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                int x = nums[i];
                int y = nums[j];
                if (x + y == target) {
                    answer_list[0] = i;
                    answer_list[1] = j;
                    break;
                }
            }
        }
        return answer_list;
    }
}

File: solution/Python/Sol1.py
from typing import List, Dict


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        answers: Dict[int, int] = {}
        i: int
        for i, num in enumerate(nums):
            x = target - num
            if x in answers:
                return [answers[x], i]
            answers[num] = i


nums = [2, 7, 11, 15]
target = 9

print("Input nums:" + "".join(str(nums)))
print("Input target:" + str(target))
print("Output:" + "".join(str(Solution().twoSum(nums, target))))
